testplan/report/testing/base.py

```diff
@@ -66,13 +68,17 @@ class Status(object):
     INCOMPLETE = 'incomplete'
```

Comment:
What does a 'incomplete' status mean?

Reply:
I don't know I didn't add it. It's used somewhere. Probably means a testcase is interrupted or not run for some reason.